docs/docs/operator-manual/environments.mdx

```diff
@@ -2,90 +2,113 @@ import ThemedImage from "@theme/ThemedImage";
 
 # Environments
 
-The Rig Platform supports a multi-environment architecture out-of-the-box.
+The Rig Platform supports a multi-environment architecture out-of-the-box. All Capsules in the Platform are
+automatically deployable to all environments. That means a single Environment always contains some sub-set of all Platform Capsules.
 
-Environments are associated to a single cluster, meaning Rig supports multiple clusters
-through environments.
-
-While any mix-and-match are supported, it essentially boils down to two variants,
-singletenancy and multitenancy. There are no limits to the number of environments,
-clusters or environments per cluster.
-
-## Singletenancy
-
-In a singletenancy setup, the given cluster hosts only a single environment. A simple
-example of this is two clusters, production and staging, that respectively hosts a
-production and a staging environment.
+Each environment has its own configuration of a given Capsule. Capsule A can e.g. have 10 replicas in `production` but only 2 in `staging`.
 
 <ThemedImage
-  alt="Singletenancy"
+  alt="Topology"
   customProps={{
     zoom: false,
   }}
   sources={{
-    light: "/img/operator-manual/environments-single-light.svg",
-    dark: "/img/operator-manual/environments-single-dark.svg",
+    light: "/img/operator-manual/environment-topology.svg",
+    dark: "/img/operator-manual/environment-topology-dark.svg",
   }}
 />
 
-In this setup, the Rig Platform communicates with both clusters, to configure the two
-environments individually. The configuration for the Rig Platform Helms values file
-looks like this:
-
-```yaml
-rig:
-  clusters:
-    staging:
-      type: k8s
-    production:
-      type: k8s
-  environments:
-    staging:
-      cluster: staging
-    production:
-      cluster: production
-      default: true
+In this example, changes made to Capsules in Environment B are not reflected to Capsules in Environment A, and vice-versa.
+
+## Creating new environments
+
+To create a new environment, use the following CLI command:
+
+```bash
+rig env create my-environment my-cluster
 ```
-:::info Default environment
-The `default` marker is used by the Dashboard and CLI for which to pick when nodes
-environment is specified, e.g. loading the dashboard will in this case show the production
-environment.
+
+The newly created environment is immediate available for existing Capsules as a deployment target:
+
+```bash
+rig deploy -E my-environment my-capsule --image nginx
+```
+
+### Custom namespace templating
+
+By default, Capsules in the new environment will be named after the project. To change this,
+a custom namespace template can be provided:
+
+```bash
+rig env create my-environment my-cluster \
+  --namespace-template '{{ .Project.Name }}-{{ .Environment.Name }}'
+```
+
+This example will suffix all namespaces with the environment, allowing multiple environments in a single
```

Comment:
Namespaces or capsules? Do we support multiple ends in the same namespace? I thought it was always one project/env per namespace